src/serialization/ObjectData.ts

```diff
@@ -191,7 +191,14 @@ export class ObjectDataOutput implements DataOutput {
         this.writeArray(this.writeShort, shorts);
     }
 
+    /** @deprecated since version 4.2.
```

Comment:
Do we need these jsdocs? I mean, they are already there in the interface, I think duplicating it here in the implementation is unnecessary. The same applies to other places

Reply:
Ok I forget this sorry, I will remove them

Reply:
Well, it turns out that my IDE does not show deprecation by marking method as strike-through when I don't put deprecation to the class that implements the interface. However, this should be ok since the classes are internal and only we use them.

Reply:
Same for me, when I explicitly declare the type with the interface, it shows it. But as you said, this is not a problem, even we don't use the concrete type apart from initializing the object.